app/assets/stylesheets/common/base/compose.scss

```diff
@@ -65,8 +65,10 @@ html.composer-open {
   }
 
   &.open {
+    --min-height: 255px;
     box-sizing: border-box;
     height: var(--composer-height);
+    min-height: var(--min-height);
```

Comment:
This got inadvertently removed in the previous change, we need this to make sure that composer doesn't have a height of `0px` set by the user inadvertently.